ibm/service/database/resource_ibm_database.go

```diff
@@ -140,8 +80,7 @@ func ResourceIBMDatabaseInstance() *schema.Resource {
 		CustomizeDiff: customdiff.All(
 			resourceIBMDatabaseInstanceDiff,
 			validateGroupsDiff,
-			validateUsersDiff,
-			validateRemoteLeaderIDDiff),
+			validateUsersDiff),
```

Comment:
Confirming that this is still working as expected now that the function is in a separate file.